test/object-store/migrations.cpp

```diff
@@ -1154,7 +1154,7 @@ TEST_CASE("migration: Automatic") {
             list.insert(0, Mixed{10});
             list.insert(1, Mixed{10.10});
             list.insert(2, Mixed{ObjLink{target_table->get_key(), target_object.get_key()}});
-            list.insert(3, Mixed{target_object.get_key()});
+            list.insert(3, Mixed{ObjLink{target_table->get_key(), target_object.get_key()}});
```

Comment:
It is not valid to have a Mixed link stored without a destination table, unless there is only one dest table for the entire property as defined in the schema.